jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/EagerFormHandler.java

```diff
@@ -61,8 +62,10 @@ public boolean handle(Request request, org.eclipse.jetty.server.Response respons
 
     protected boolean handleFormFields(Request request, org.eclipse.jetty.server.Response response, Callback callback)
     {
-        Request.Handler handler = super::handle;
-        Promise<Fields> onFields = new Promise<>()
+        Request.Handler handler = getHandler();
+        InvocationType invocationType = handler.getInvocationType();
+        AtomicBoolean immediate = new AtomicBoolean(true);
```

Comment:
I'd rename the variable to `done` and invert the true/false, but it's a small nit.

Reply:
counting down from an atomic integer of 2 is better